packages/cli/src/editor/components/TimelineDragHandler.tsx

```diff
@@ -1,33 +1,27 @@
 import React, {useCallback, useState} from 'react';
 import {Internals, interpolate} from 'remotion';
 import styled from 'styled-components';
-import {
-	TIMELINE_LEFT_PADDING,
-	TIMELINE_RIGHT_PADDING,
-} from '../helpers/timeline-layout';
+import {TIMELINE_PADDING} from '../helpers/timeline-layout';
 import {useWindowSize} from '../hooks/use-window-size';
 
 const Container = styled.div`
+	flex: 1;
 	position: relative;
-	padding-left: ${TIMELINE_LEFT_PADDING}px;
-	padding-right: ${TIMELINE_RIGHT_PADDING}px;
-	padding-top: 20px;
-	margin-top: -10px;
+	padding: ${TIMELINE_PADDING}px;
```

Comment:
We replaced padding-left and padding-right with padding on all 4 sides. Was that intentional?

Reply:
Yes, it was. From what I've tested, there was no need for different padding. It changes the layout a bit but I think it looks good. Do you prefer to revert it?

Reply:
OK, then LGTM!